pkg/dashboard/store/metrics/metrics.go

```diff
@@ -43,6 +42,7 @@ type Collector struct {
 // NewCollector initializes metrics and collector
 func NewCollector(experimentStore core.ExperimentStore, scheduleStore core.ScheduleStore, workflowStore core.WorkflowStore) *Collector {
 	return &Collector{
+		log:             NewDefaultZapLogger(),
```

Comment:
please do not instant new logger inside of the constructor, acquire it as one of constructor parameter.